src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -103,25 +119,36 @@ function BaseHeader(props: HeaderProps) {
   const icons: { [key: string]: JSX.Element } = {
     backNav: getIcon(backAction || backActionDefault, <ArrowBackIcon />),
     edit: getIcon(navigateToEdit, <CreateIcon />, true),
-    user: getIcon(openProfileMenu, <AccountCircleIcon className={classes.account} fontSize="large" />),
+    user: getIcon(openProfileMenu, <MenuIcon className={classes.account} fontSize="large" />),
   };
 
   const left = leftIcon ? icons[leftIcon] : null;
   const header = title ? (
-    <Typography className={classes.title} variant="h2">
+    <Typography className={searchAction ? classes.leftTitle : classes.title} variant="h2">
       {title}
     </Typography>
   ) : null;
   const right = rightIcon ? icons[rightIcon] : null;
 
+  const onSearchExit = () => {
+    setSearchVisibility(false);
+    searchExit();
+  }
+
   return (
-    <div className={classes.root}>
+    <div className={classes.root} style={{ marginTop: props.searchAction ? '20px' : undefined }}>
       {header}
       <div className={classes.toolbar}>
         <div className={classes.left}>{left}</div>
-        <div className={classes.right}>{right}</div>
+        <div className={classes.right}>
+          {props.searchAction ? getIcon(() => setSearchVisibility(true), <SearchIcon className={classes.search} />) : null}
```

Comment:
Fun fact: you can use `&&` and `||` in place of ternaries when you only have one case!

So instead of falling back to `null`, you can make this 
```
{props.searchAction && getIcon(() => setSearchVisibility(true), <SearchIcon className={classes.search} />)}
```